stock_orderpoint_mto_as_mts/models/product_template.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        template = super().create(vals_list)
+        template.product_variant_ids.sudo()._ensure_default_orderpoint_for_mto()
+        return template
```

Comment:
If we drop this, will fail on case: Create product.template --> Create orderpoints

Because when create template --> template will trigger [here](https://github.com/odoo/odoo/blob/e3033d7aae5537403a1aedbae6e598514c04f703/addons/product/models/product_template.py#L433) --> for now, products will be created but `is_mto` in not computed (debugged) so we can not handle this case through `def create() of product.product`